tests/test99.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include "utarray.h"
+
+int main(void) {
+    UT_array *nums;
+    int i, *p;
+
+    utarray_new(nums,&ut_int_icd);
+    for(i=0; i < 10; i++) utarray_push_back(nums,&i);
+
+    i = 110;
+    utarray_replace(nums, 5, &i);
```

Comment:
One thing I'd be slightly worried about people misunderstanding about a "replace" API is that the `5` here is an index, not a value. To that end, I suggest that this test should initialize the array (on line 9) with values that don't match their indices. E.g.

    for (i = 1; i < 1024; i <<= 1) {
        utarray_push_back(nums, &i);
    }
    i = 110;
    utarray_replace(nums, 5, &i);

(Note the whitespace/brace style, while you're at it, please.)